pom.xml

```diff
@@ -600,7 +601,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-release-plugin</artifactId>
-                    <version>2.5.3</version>
+                    <version>${maven.release.version}</version>
```

Comment:
❔ How does this related to maven repository caching? Or are you improving releases in this PR as well?

Reply:
Looks like the PR also updates the Maven version used in the builds. I imagine this is part of that change.

Reply:
That makes sense so this will version of the release plugin works better with the latest version of Maven, gotcha 👍